apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -390,3 +390,32 @@ export const Customization: Story = {
         );
     }
 }
+
+export const Formats: Story = {
+    args: {
+        width: 500,
+        height: 600,
+        dayHeaderFormat: 'shortdate',
```

Comment:
I would prefer to provide the ability to choose from several values that are appropriate for each option. This would help avoid questions like, "What format do I need to enter, and did I enter it correctly?" and "What format options are available?"
```
 argTypes: {
        dayHeaderFormat: {
            control: 'select',
            options: ['shorttime', 'hh:mm:ss', 'longdate', 'none'],
            mapping: {
                ['shorttime']: 'shorttime',
                ['hh:mm:ss']: 'hh:mm:ss',
                ['longdate']: 'longdate',
                ['none']: null,
            },
            defaultValue: 'shorttime',
        }
    },
    ```
This is a draft for demonstration purposes only. Please consider which formats we can offer as selectable options.

Reply:
3bce5ead833536bf44730662f11c3efb81a06471

Reply:
im not sure why u decided to merge this, i'd rather expect only time formats for `messageTimestampFormat` and only date for `dayHeaderFormat`. But let keep this at it is since u allready disscused it with Eugeniy


and it would be good to add some custom formats like `d of MM, yyyy` and `hh_mm` or smth